From 44d1022cc1134d17e1e381fca4761c6e506a4f3f Mon Sep 17 00:00:00 2001
From: mari <maricantdecide@gmail.com>
Date: Fri, 24 May 2024 17:34:44 -0300
Subject: [PATCH] Allow replacing the login profile through LoginEvent.


diff --git a/proxy/src/main/java/net/md_5/bungee/connection/InitialHandler.java b/proxy/src/main/java/net/md_5/bungee/connection/InitialHandler.java
index 174098a7..7328e41e 100644
--- a/proxy/src/main/java/net/md_5/bungee/connection/InitialHandler.java
+++ b/proxy/src/main/java/net/md_5/bungee/connection/InitialHandler.java
@@ -674,6 +674,7 @@ public class InitialHandler extends PacketHandler implements PendingConnection
                     return;
                 }
 
+                loginProfile = result.getLoginResult(); // Zartema
                 userCon = new UserConnection( bungee, ch, getName(), InitialHandler.this );
                 userCon.setCompressionThreshold( BungeeCord.getInstance().config.getCompressionThreshold() );
 
-- 
2.43.0

